presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestSqlStandardAccessControlChecks.java

```diff
@@ -94,12 +94,14 @@ public void testAccessControlCreateTableAsSelect()
     {
         String createTableAsSelect = "bob_create_table_as_select";
 
+        bobExecutor.executeQuery("DROP TABLE IF EXISTS " + createTableAsSelect);
         assertThat(() -> bobExecutor.executeQuery(format("CREATE TABLE %s AS SELECT * FROM %s", createTableAsSelect, tableName)))
                 .failsWithMessage(format("Access Denied: Cannot select from table default.%s", tableName));
 
         aliceExecutor.executeQuery(format("GRANT SELECT ON %s TO bob", tableName));
         bobExecutor.executeQuery(format("CREATE TABLE %s AS SELECT * FROM %s", createTableAsSelect, tableName));
         assertThat(bobExecutor.executeQuery(format("SELECT * FROM %s", createTableAsSelect))).hasNoRows();
+        bobExecutor.executeQuery("DROP TABLE " + createTableAsSelect);
```

Comment:
Why don't we drop this table in a `finally` block?

Reply:
Because leaving it behind is insignificant harm -- we `DROP IF EXISTS` at the beginning, so re-running the test will work OK.
(`finally` incurs some readability penalty, so i prefer not to use it, if it is not needed)